src/theme/page.scss

```diff
@@ -1,11 +1,27 @@
+html {
+    height: 100vh;
+    width: 100vw;
+}
+
+body {
+    height: 100vh;
+    width: 100vw;
+    display: flex;
+}
+
 #stac-browser {
+    display: flex;
+    flex-direction: column;
     max-width: 100%;
+    overflow: auto;
```

Comment:
For screens with resolutions larger than Full HD, you get inline scrollbars, which I think is not ideal:

![grafik](https://user-images.githubusercontent.com/8262166/175830328-14dcbde9-078e-4d3d-9ae9-31f953bbeee5.png)

Reply:
Changing the CSS as follows seems to work for me:

```css
html {
    height: 100%;
    width: 100%;
}

body {
    height: 100%;
    width: 100%;
    display: flex;
}

#stac-browser {
    display: flex;
    flex-direction: column;
    max-width: 100%;

...
```

Does that make sense?